tests/unit/parsers/test.manifestjson.js

```diff
@@ -5909,5 +5909,85 @@ describe('ManifestJSONParser', () => {
       ]);
       expect(manifestJSONParser.isValid).toEqual(true);
     });
+
+    it('emits an error when "none" is specified with other data collection permissions', () => {
```

Comment:
thought: Given that we had to workaround the import of the schema definitions for the data collection permissions more than we would have liked to, it may be also worth to consider if adding another test case to verify that we emit a lintint error as expected when "technicalAndInteraction" is wrongly listed in the required permission, wdyt?